### DISCUSSION TITLE
Video Demo fin Sprint

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#273] [Ideas]
Abro esta discusión para que nos organicemos en cuanto a el tema que va a hablar cada uno en el video final.

**Comment by NataliaAlvarezIspc** [comment]
Voy a hablar de Registración... También puedo hablar del login

**Comment by BraianTroncoso** [comment]
Voy a hablar del back con usuarios y la conexión con el front

**Comment by MarielaGomez** [comment]
Puedo hablar de la navegabilidad- formulario de contacto- presentación del proyecto

**Comment by Valentino399** [comment]
Puedo hablar acerca del diseño hecho en Figma

**Comment by erick-lpz** [comment]
Voy a hablar sobre el dashboard

**Comment by yan0106** [comment]
Puedo hablar sobre el perfil del usuario y su actualización. También eliminar cuenta.

**Comment by dfg259** [comment]
Puedo hablar de algunas cuestiones que trabajé en el perfil y el video que va en quienes somos.
